apps/demos/Demos/DataGrid/CustomEditors/Vue/EmployeeDropDownBoxComponent.vue

```diff
@@ -57,6 +57,10 @@ const currentValue = ref(props.value);
 const dropDownBoxRef = ref<DxDropDownBox | null>(null);
 const dropDownOptions: DxDropDownBoxTypes.Properties['dropDownOptions'] = { width: 500 };
 
+const getSelectedRowKeys = (value: any): any[] => {
+  return value ? [value] : [];
```

Comment:
I think there is wrong result in the case if value === 0. The method will return [] instead of [0].